Gallery update

will-moore opened the pull request:
As discussed at IDR meeting this morning:
 - /gallery/ will become the home-page of IDR (see https://github.com/ome/omero-gallery/pull/31).
 - https://idr.openmicroscopy.org/about/ will include text from https://idr.openmicroscopy.org/about/about.html
 - The "About Us > Overview" link in header links to https://idr.openmicroscopy.org/about/index.html.
 - The "Take a look at the data" orange button links to /gallery/ home page"

sbesson commented on index.html:
```diff
@@ -10,7 +10,7 @@
                     <h2 class="hero-subheader small-10 medium-10 large-10 small-offset-1 medium-offset-1 large-offset-1">Welcome to the Image Data Resource (IDR). This online, public data repository seeks to store, integrate and serve image datasets from published scientific studies.</h2>
                     <br>
-                    <a href="/webclient/userdata/?experimenter=-1" class="large button sites-button">Take a look at the data</a>
+                    <a href="/gallery/" class="large button sites-button">Take a look at the data</a>
```
You will need to update https://github.com/IDR/idr.openmicroscopy.org/blob/master/build.sh#L8 to ignore `/gallery` in the linkcheck step

will-moore commented:
@sbesson That's strange: when I initially compared font-sizes on the idr vv gallery, I also saw some difference, but it turned out for me that I had the gallery page zoomed in to 110%. With ```Ctrl -``` I reset to 100% and now they look the same.

Also added Cell-IDR and Tissue-IDR links and moved Analysis and Deployment links under 'About' menu on the right of the page, as suggested in https://github.com/ome/omero-gallery/issues/26#issuecomment-499144751

sbesson commented:
Thanks @will-moore for aligning the styles. I also appreciate the reduction in the number of top-level elements. Proposing to get this tagged and deployed with gallery together with the redirects i.e. in a production like environment. From there, we can review the outstanding styling issues and capture RFEs as issues.
